packages/react-storage/src/components/StorageBrowser/views/LocationsView/useLocationsView.ts

```diff
@@ -108,9 +110,8 @@ export function useLocationsView(
         options: { ...listOptions, refresh: true },
       });
     },
-    onPaginateNext: () => {
-      handlePaginateNext({ resultCount, hasNextToken });
+    onPaginate: (newPage: number) => {
+      handlePaginate(newPage);
     },
```

Comment:
Can rename `handlePaginate` to `onPaginate` and pass it directly here

Reply:
True. I renamed the function to `onPaginate`, but retained the callback, renaming it from `onPaginate` to `paginateCallback`. Let me know if this is in line with what you had in mind